siim_andy_tcga-50-5072/Condition/condition-5072.json

```diff
@@ -0,0 +1,58 @@
+{
+  "id": "a411079800582267",
+  "resourceType": "Condition",
+  "text": {
+    "status": "generated",
+    "div": "<div xmlns=\"http://www.w3.org/1999/xhtml\">Whole Body Tc-99m Bone Scintigraphy 2/23/2000</div>"
```

Comment:
I'm going to merge this, and fix a few issues.  For the purposes of your code - you will need to generate a <div> that matches the content of the condition.  This is currently a valid FHIR object, but the content of the div doesn't match the rest of the resource.